src/main/java/com/minecolonies/coremod/colony/jobs/views/DefaultJobView.java

```diff
@@ -26,9 +27,9 @@
     private final IColonyView colonyView;
 
     /**
-     * The synched job name.
+     * The synched job registry entry.
      */
-    private String name;
+    private JobEntry entry;
```

Comment:
We probably also should give the non-view jobs their entry object instead of hardcoding the return, to at least be consistent with job/jobview

Reply:
Can do in the next PR, feel like there is even more things we can do justifying more PRs =D

Reply:
well it'd be part of your description of "Give jobs and views their entry" :D